stankove_napady.md

```diff
@@ -0,0 +1,105 @@
+Zkusim popsat svuj navrh a zkusme to spolecnymi pripominkami upravit do lepsi podoby. Je to moje stankova premiera.
+<br>
+</br>
+## Co chceme prezentovat?
+
+**python.cz** 
+
+> aby je navstevovalo vice pythonistu i zacatecniku zajemcu o python
+
+> aby vedeli jake informace tam najdou
+
+**pyvo.cz**
+
+> protoze lide maji vedet, ze kazda zasadni zmena okolo mista ci casu srazu by se mela jako prvni objevit predevsim tam? tzn. naucit lidi hledat ze vsech kanalu dulezito info prave tady
+
+**PyLadies a Advanced srazy**
+
+> zvysit povedomi o PyLadies celkove (stale se setkavam jako organizatorka s tim, ze spousty python lidi o teto aktivite lidi z komunity vubec nevi)
+
+> zvysit povedomi i u mensich firem, ktere by to jakkoliv rady podporily
+
+> moznost jak zapojit zase nove kouce, kteri by napriklad opravdu radi Python ucili o moznosti na PyLadies nevi
+
+**naucse.python.cz**
+
+> otazka na @encukou  zdali jsme schopni za mesic posunou tuto novou vec tak, abychom ji tam mohli prezentovat?
+
+> vidim to jako velmi zajimave doplneni stanku vedle PyLadies - bude pekne videt, ze mame zajem ucit vsechny bez rozdilu
+
+**Zapojse a 📚  blog.python.cz**
+
+> to by tam nemelo chybet
+
+**PyCon?**
+> ze bude?
+
+<br>
+</br>
+## Kdo to muze prezentovat - sonda do vaseho casoveho rozvrhu
+je mi velmi jasne, ze vetsina z nas rada uslysi prednasky. Az tedy na me, ja se na ne tolik nehrnu. Bylo by lepsi dat si dopredu jak se da s vami na stanku pocitat ci zdali bych si s sebou mela nekoho vzit z holek - ale nemyslim si, ze je tolik dobry napad, aby tam byly jen holky - mozna by mel zajem @stlk?
+
+| Kdo        | Kdy           | Jak muzu pomoci |
+| ------------- |:-------------:| -----:|
+| Veronika    | klidne celou dobu | muzu povidat skoro o vsem - nemam ale tolik co rict k zapojse a naucse |
+|    ?   |       |    |
+|   ?    |       |    |
+
+<br>
+</br>
+## Co by na stanku nemelo chybet
+
+> v prvni rade, dotaz organizatoru je, zdali potrebujeme LAN  (wifi jinak k dispozici bude) a zdali potrebujeme zasuvky, toto je zrejme otazka na @encukou, zatim ji neumim sama rozhodnout
+<br>
+</br>
+**1. fotky z akci vytistene, hlavne hipsta fotky s ESPckem a hipsa s UKULELE** @lspdv zaridi slozku, kde vybereme, necha vytisknout, meli bychom na to mit k dispozici nejake nastenky, dohodnu v organizatorama
+
+**2. video z Pyconu nebo videa z prednasek z Pyva** @encukou a dulezite pro @lspdv vedet na cem to chceme spoustet
+
+**3. samolepy PyLadies** @lspdv zaridi tisk pr. tiskovou pripravu
+
+> navrhuji udelat podstatne mensi a jen cervene bez hlav Máni
+
+> zvazuji udelat navrh malych, kde bude napis pyladies a neco ve stylu "I support girls who love to code in Python - touzim po tom, aby lidi vedeli ze samolepky o co jde, ciste pyladies je hrozne nudo
+
+**4. samolepky python.cz** @honzajavorek navrh, @lspdv zaridi tisk pr. tiskovou pripravu
```

Comment:
udelejte mi na to prosim zvlast issue a ja se tam zkusim soustredit na to, at to v terminu dodelam

chceme samolepky, http://www.badgees.cz/, nebo oboje? co se tyce obsahu, muzu zkusit nastrelit nejaky navrhy a pak se rozhodnem co chceme zpracovat ... souvisi s #12